packages/react-router-dom-v5-compat/__tests__/compat-router-test.tsx

```diff
@@ -0,0 +1,30 @@
+/**
+ * @jest-environment node
+ */
```

Comment:
If there is a better/preferred way to set the environment in jest, please let me know 🙂. Without this comment this test will have a false-positive, since it assumes we are in a browser environment where this isn't a problem.